presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -92,6 +92,35 @@
      */
     Optional<PartitioningHandle> getCommonPartitioning(Session session, PartitioningHandle left, PartitioningHandle right);
 
+    /**
```

Comment:
I wonder if it makes sense to copy this long comment. What if we leave it only in the SPI in the  `ConnectorMetadata` class?

Reply:
@arhimondr : I agree we should only keep in one place and have a de-briefed version in other place. Just don't know where should be the "full" version be. 

Any suggestions @rschlussel  ? :)